internal/engine/test.go

```diff
@@ -50,10 +52,15 @@ func (t *test) report(res *http.Response) {
 		result = "FAIL"
 	}
 
-	fmt.Printf("%s : %s\n", result, t.Url)
+	fmt.Printf("%s : %s %s\n", result, t.Method, t.Url)
 	if len(errors) > 0 {
 		for _, e := range errors {
 			fmt.Printf("  %s\n", e)
 		}
+		code := len(errors)
+		if code > 125 {
+			code = 125
+		}
+		os.Exit(code)
```

Comment:
Adding comments to explain the rationale behind the exit code handling would make this even better.

Reply:
will address better documentation in future PR